doc/api/permissions.md

```diff
@@ -440,7 +446,157 @@ not adopt the origin of the `blob:` URL.
+the capabilities to access the filesystem, spawn process and,
+use `node:worker_threads` will be restricted.
+
+```console
+$ node --experimental-permission index.js
+node:internal/modules/cjs/loader:171
+  const result = internalModuleStat(filename);
+                 ^
+
+Error: Access to this API has been restricted
+    at stat (node:internal/modules/cjs/loader:171:18)
+    at Module._findPath (node:internal/modules/cjs/loader:627:16)
+    at resolveMainPath (node:internal/modules/run_main:19:25)
+    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:24)
+    at node:internal/main/run_main_module:23:47 {
+  code: 'ERR_ACCESS_DENIED',
+  permission: 'FileSystemIn'
+}
+```
+
+Allowing access to spawning a process and creating worker threads can be done
+using the `--allow-child-process` and `--allow-worker` respectively.
+
+#### Runtime API
+
+When enabling the Permission Model through the [`--experimental-permission`][]
+flag a new property `permission` is added to the `process` module.
+This property contains two functions:
+
+* `permission.deny(scope [,parameters])`
+  * `scope` {string}
+  * `parameters` {Array} array of strings
+  * Returns: {boolean} Operation result.
+
+API Call to _deny_ permissions at runtime. e.g
+
+```js
+process.permission.deny('fs'); // Deny permissions to ALL fs operations
+
+// Deny permissions to ALL FileSystemOut operations
+process.permission.deny('fs.write');
+// deny FileSystemOut permissions to the protected-folder
+process.permission.deny('fs.write', ['/home/rafaelgss/protected-folder']);
+// Deny permissions to ALL FileSystemIn operations
+process.permission.deny('fs.read');
+// deny FileSystemIn permissions to the protected-folder
+process.permission.deny('fs.read', ['/home/rafaelgss/protected-folder']);
+```
+
+* `permission.check(scope [,parameters])`
+  * `scope` {string}
+  * `parameters` {string}
+
+API Call to _check_ permissions at runtime.
+
+```js
+process.permission.check('fs.write'); // true
+process.permission.check('fs.write', '/home/rafaelgss/protected-folder'); // true
+
+process.permission.deny('fs.write', '/home/rafaelgss/protected-folder');
+
+process.permission.check('fs.write'); // true
+process.permission.check('fs.write', '/home/rafaelgss/protected-folder'); // false
+```
+
+#### File System Permissions
+
+To allow access to the filesystem, you need to use the flag `--allow-fs`
+
+```console
+$ node --experimental-permission --allow-fs=read index.js
+Hello world!
+(node:19836) ExperimentalWarning: Permission is an experimental feature
+(Use `node --trace-warnings ...` to show where the warning was created)
+```
+
+The valid arguments for the `--allow-fs` flag option are:
+
+* `write` - To manage `FileSystemOut` (Writing) operations.
+* `read` - To manage `FileSystemIn` (Reading) operations.
+* `fs` - To allow all the `FileSystem` operations.
+
+Example:
+
+* `--allow-fs=read:/tmp/` - It will allow `FileSystemIn` access to the `/tmp/`
```

Comment:
There seems to be a lot of overlap with the docs added to cli.md. I think we should be careful about duplicating content because it can get out of sync. We can link to the appropriate docs when necessary instead.